### PR TITLE
Updated readme to new architecture and added b3dm instructions.

### PR DESCRIPTION
**Pull request by Jeremy-Gaillard**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -42,28 +42,51 @@ If you wan to run unit tests:
...
-- *FeatureTable* with a header *FeatureTableHeader* and a *FeatureTableBody*
-- *FeatureTableBody* which contains features of type *Feature*
+- *TileHeader* represents the metadata of the tile (magic value, version, ...)
+- *TileBody* contains varying semantic and geometric data depending on the the tile's type
 
-Moreover, a utility class *TileReader* is available to read a *.pnts*
+Moreover, a utility class *TileReader* is available to read a tile
 file as well as a simple command line tool to retrieve basic information
-about a Point Cloud file **py3dtiles\_info**.
+about the tile **py3dtiles\_info**.
```
Comment (elemoine) [inline README.rst]: How about adding a column after "tile" here? Also, I'd use "about a tile" rather than "about the tile".

File: README.rst [hunk 2]
```diff
@@ -42,28 +42,51 @@ If you wan to run unit tests:
...
+
+**How to use py3dtiles\_info**
+
+If we want to retrieve basic information about the file *pointCloudRGB.pnts*:
```
Comment (elemoine) [inline README.rst]: I'd start the sentence with "For example,".

File: README.rst [hunk 3]
```diff
@@ -42,28 +42,51 @@ If you wan to run unit tests:
...
+    Feature table json byte length:  148
+    Feature table bin byte length:  15000
+
+    Feature Table Header
+    --------------------
+    {'POSITION': {'byteOffset': 0}, 'RGB': {'byteOffset': 12000}, 'POINTS_LENGTH': 1000, 'RTC_CENTER': [1215012.8828876738, -4736313.051199594, 4081605.22126042]}
+
+    First point
+    -----------
+    {'Z': -0.17107764, 'Red': 44, 'X': 2.19396, 'Y': 4.4896851, 'Green': 243, 'Blue': 209}
```
Comment (elemoine) [inline README.rst:81]: The above code block is not rendered on GitHub.  I think it's important that the README file renders correctly on GitHub.

**Review by elemoine — CHANGES_REQUESTED** [review]

File: README.rst (hunk 3 above)
Comment (pblottiere) [reply README.rst:81]: FWIW, I used the code block syntax in order to correctly render the README on pypi (I think Markdown is not supported).

Comment (elemoine) [reply README.rst:81]: Yes, but other code blocks are rendered on GitHub. Only that particular one is not. Maybe the missing blank line after the `.. code-block: shell` declaration.

File: README.rst (hunk 1 above)
Comment (Jeremy-Gaillard) [reply README.rst]: What do you mean by "adding a column"?

Comment (Jeremy-Gaillard) [reply README.rst]: Oh, do you mean a colon, the ":" punctuation mark? Adding it makes sense, I'll do it.

**Comment by Jeremy-Gaillard** [comment]
I applied the requested change in the amended commit.

File: README.rst (hunk 1 above)
Comment (elemoine) [reply README.rst]: Sorry, I meant adding a colon (`:`).

**Review by elemoine — APPROVED** [review]

**Comment by elemoine** [comment]
Please merge when you think this is ready.

**Pull request merged**